packages/form-js-viewer/src/render/components/index.js

```diff
@@ -12,6 +13,7 @@ export {
   Button,
   Checkbox,
   Default,
+  Fieldset,
```

Comment:
List is alphabetically ordered here as well.

Reply:
I don't get you here, `Fieldset` has been alphabetically ordered.

Reply:
I don't get myself either to be honest ❓ 

Sorry about that 20 hours ago me must have missed his coffee 😄